repository-data/site/src/main/resources/hcm-content/hst/configurations/lks/workspace/pages/record-number-of-gps-contentpage.yaml

```diff
@@ -0,0 +1,36 @@
+/hst:hst/hst:configurations/lks/hst:workspace/hst:pages/record-number-of-gps-contentpage:
```

Comment:
Could we move this under `repository-data/site-development` (as it is a development data) ?

Reply:
Done.